lib/utils/acars_free_text.ts

```
@@ -0,0 +1,4 @@
+export function formatAcarsFreeText(text?: string | null): string {
+  if (!text) return '';
+  return text.trim().replace(/\s+/g, ' ');
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)